speech/tests/system/gapic/v1/test_system_speech_v1.py

```diff
@@ -12,25 +12,77 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import time
+
+import os
+import io
 
 from google.cloud import speech_v1
-from google.cloud.speech_v1 import enums
-from google.cloud.speech_v1.proto import cloud_speech_pb2
 
+BUCKET = os.environ["GOOGLE_CLOUD_TESTS_SPEECH_BUCKET"]
 
 class TestSystemSpeech(object):
+
     def test_recognize(self):
 
         client = speech_v1.SpeechClient()
-        language_code = "en-US"
-        sample_rate_hertz = 44100
-        encoding = enums.RecognitionConfig.AudioEncoding.FLAC
+
         config = {
-            "language_code": language_code,
-            "sample_rate_hertz": sample_rate_hertz,
-            "encoding": encoding,
+            "encoding": speech_v1.enums.RecognitionConfig.AudioEncoding.FLAC,
+            "language_code": "en-US",
+            "sample_rate_hertz": 16000    
         }
-        uri = "gs://gapic-toolkit/hello.flac"
+
+        uri = "gs://{}/broonklyn.flac".format(BUCKET)
         audio = {"uri": uri}
+
         response = client.recognize(config, audio)
+        assert response.error.code == 200
+
+
+    def test_longRunningRecognize(self):
+
+        client = speech_v1.SpeechClient()
+
+        config = speech_v1.types.RecognitionConfig(
+            encoding = speech_v1.enums.RecognitionConfig.AudioEncoding.FLAC,
+            language_code = "en-US",
+            sample_rate_hertz = 16000)
+
+        uri = "gs://{}/broonklyn.flac".format(BUCKET)
+        audio = speech_v1.types.RecognitionAudio(uri)
+                  
+        
+        response = client.long_running_recognize(config=config, audio=audio)
+        assert response.error.code == 200
+
+
+    def test_streamingRecognize(self):
+
+        client = speech_v1.SpeechClient()
+                  
+        config = speech_v1.types.RecognitionConfig(
+            encoding = speech_v1.enums.RecognitionConfig.AudioEncoding.FLAC,
+            language_code = "en-US",
+            sample_rate_hertz = 16000)
+        streamingConfig = speech_v1.types.StreamingRecognitionConfig(config = config)
+
+        uri="gs://{}/broonklyn.flac".format(BUCKET)
+        with io.open(uri,'rb')as stream:
+            requests = [speech_v1.types.StreamingRecognizeRequest(
+                 audio_content=stream.read()
+            )]
+        
+        response = client.streaming_recognize(streamingConfig, requests)
+        assert response.error.code == 200    
+
+    
+if __name__ == "__main__":
```

Comment:
Please remove this section. We use pytest to run the tests.

```pytest tests/system/gapic/v1/test_system_speech_v1.py```

Alternatively you can use [`nox`](https://nox.thea.codes/en/stable/) to run the tests. It will create a fresh virtual environment and install the necessary dependencies before running the tests.

```nox -s system```